site/plugins.json

```diff
@@ -615,7 +615,7 @@
     "name": "Next.js ",
     "package": "@netlify/plugin-nextjs",
     "repo": "https://github.com/netlify/next-runtime",
-    "version": "4.41.3",
+    "version": "5.11.2",
```

Comment:
Change forced by test `Plugin version is the same as the first non-feature-flagged compatibility version`

Reply:
Ref for this change of top-level version not making practical difference
https://github.com/netlify/build/blob/ead889f009f44154b34b43fdb2e86b01cf3cb796/packages/build/src/plugins/list.ts#L133

this `version` is only used when there is no `compatibility` list (and there is one for next, hence it's not being actually used).

The test that forced change of top-level version seems to try to enforce convention rather than ensuring correct behaviors